src/abbfreeathome/devices/cover_actuator.py

```diff
@@ -62,9 +72,9 @@ def __init__(
         )
 
     @property
-    def state(self) -> int | None:
+    def state(self) -> CoverActuatorState | None:
```

Comment:
The return here would be a string and it would never be `None.

```suggestion
    def state(self) -> str:
```